_locales/en/messages.json

```diff
@@ -458,9 +458,6 @@
 	"summary_description_not_scanned_2": {
 		"message": "Navigate to another page and I promise I'll deliver the goods."
 	},
-	"summary_custom_settings": {
```

Comment:
I'm seeing that this token is used in `GhosteryFeature.jsx`

Reply:
The token in `GhosteryFeature.jsx` was in some commented out code that I had left in there and which is now removed. The remaining code does not use this token.